tests/ethereum/ethereum.spec.ts

```diff
@@ -0,0 +1,665 @@
+import { impersonateAddress } from "./../../helpers/utilities/fork";
+import { AaveEcosystemReserveController__factory } from "./../../typechain/factories/@aave/periphery-v3/contracts/treasury/AaveEcosystemReserveController__factory";
+import { Ownable } from "./../../dist/types/typechain/@aave/core-v3/contracts/dependencies/openzeppelin/contracts/Ownable.d";
+import { getTreasuryAddress } from "./../../helpers/market-config-helpers";
+import {
+  getERC20,
+  getFlashLoanLogic,
+  getOwnableContract,
+} from "./../../helpers/contract-getters";
+import {
+  getEthersSigners,
+  getFirstSigner,
+} from "./../../helpers/utilities/signer";
+import { AaveEcosystemReserveV2__factory } from "./../../typechain/factories/@aave/periphery-v3/contracts/treasury/AaveEcosystemReserveV2__factory";
+import {
+  INCENTIVES_PROXY_ID,
+  TREASURY_CONTROLLER_ID,
+  TREASURY_IMPL_ID,
+  TREASURY_PROXY_ID,
+} from "./../../helpers/deploy-ids";
+import { expect } from "chai";
+import { BigNumber } from "ethers";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import {
+  aave,
+  ATOKEN_IMPL_ID,
+  ConfigNames,
+  DELEGATION_AWARE_ATOKEN_IMPL_ID,
+  ETHEREUM_SHORT_EXECUTOR,
+  getAaveOracle,
+  getAaveProtocolDataProvider,
+  getACLManager,
+  getAToken,
+  getEthersSignersAddresses,
+  getIncentivesV2,
+  getIRStrategy,
+  getPool,
+  getPoolAddressesProvider,
+  getPoolAddressesProviderRegistry,
+  getPoolConfiguratorProxy,
+  getStableDebtToken,
+  getUiIncentiveDataProvider,
+  getUiPoolDataProvider,
+  getVariableDebtToken,
+  getWrappedTokenGateway,
+  ICommonConfiguration,
+  INCENTIVES_V2_IMPL_ID,
+  loadPoolConfig,
+  PoolAddressesProvider,
+  POOL_CONFIGURATOR_IMPL_ID,
+  POOL_IMPL_ID,
+  STABLE_DEBT_TOKEN_IMPL_ID,
+  strategyWETH,
+  VARIABLE_DEBT_TOKEN_IMPL_ID,
+  ZERO_ADDRESS,
+} from "../../helpers";
+
+// Prevent error HH9 when importing this file inside tasks or helpers at Hardhat config load
+declare var hre: HardhatRuntimeEnvironment;
+
+const InitializeError = "Contract instance has already been initialized";
+
+describe("Ethereum V3 - Token-less deployment", function () {
+  let addressesProvider: PoolAddressesProvider;
+  let poolConfig: ICommonConfiguration;
+  before(async () => {
+    expect(process.env.MARKET_NAME).equal("Ethereum");
+
+    if (process.env.FORK_DEPLOY === "true") {
+      await hre.deployments.fixture([
+        "market",
+        "periphery-post",
+        "after-deploy",
+      ]);
+    }
+
+    addressesProvider = await getPoolAddressesProvider();
+    poolConfig = loadPoolConfig((process.env.MARKET_NAME || "") as ConfigNames);
+  });
+
+  describe("Reserves", () => {
+    it("Market should not contain any reserves", async () => {
+      const pool = await getPool();
+
+      const assets = await pool.getReservesList();
+      expect(assets.length).equal(0);
+    });
+  });
+
+  describe("Implementations check-list", async () => {
+    it("Pool implementation should be initialized", async () => {
```

Comment:
missing checks for REVISION

Reply:
is this something that should be hardcoded? could you share your reasoning?

Reply:
I expect to have 1 as the value for REVISION on a fresh deployment. We can hardcode 1 for these tests (they are only for ethereum)

Reply:
Lets do this in another PR due it needs changes outside of this package